helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -182,6 +197,11 @@ def create(self, vals_list):
         for vals in vals_list:
             if vals.get("number", "/") == "/":
                 vals["number"] = self._prepare_ticket_number(vals)
+            # Auto-assign user
+            if "user_id" not in vals:
+                defaults = self.default_get(["user_id"])
+                if defaults.get("user_id"):
+                    vals["user_id"] = self.env.user.id
```

Comment:
Why not? It’s the same but more intuitive.

```suggestion
                if defaults.get("user_id"):
                    vals["user_id"] = defaults.get("user_id")
```